### PR TITLE
Update odo CLI reference.

### PR DESCRIPTION
**Pull request by yhontyk**
Branch 4.4, the changes are aligned with the [odo Release Notes](https://github.com/openshift/openshift-docs/pull/20957) which have been reviewed by the QE and SMEs. :blush:

### CODE REVIEW

File: modules/openshift-developer-cli-basic-odo-cli-commands.adoc [hunk 1]
```diff
@@ -395,14 +419,19 @@ Perform storage operations.
...
   odo storage delete mystorage
   
   # Delete storage mystorage from component 'mongodb'
   odo storage delete mystorage --component mongodb
+  
   # List all storage attached or mounted to the current component and
   # all unattached or unmounted storage in the current application
   odo storage list
+  
+  # Sse the` -o json` flag to get a JSON formatted output
```
Comment (bergerhoffer) [inline modules/openshift-developer-cli-basic-odo-cli-commands.adoc:433]: * Sse -> Set
* Fix the backtick so it's properly around "-o json"

**Review by bergerhoffer — COMMENTED** [review]
Just one set of typos, otherwise LGTM!

File: modules/openshift-developer-cli-basic-odo-cli-commands.adoc (hunk 1 above)
Comment (boczkowska) [reply modules/openshift-developer-cli-basic-odo-cli-commands.adoc]: @bergerhoffer fixed! Thank you!

**Comment by bergerhoffer** [comment]
Thanks for the updates, LGTM! Merging.

**Pull request merged**